### ISSUE TITLE
how many epoches needed to make the net2 converge?

### ISSUE THREAD

**Issue by ArtemisZGL** [andabi/deep-voice-conversion#109]
after training 350 epoches, my loss rate is still 0.023, but somebody told me it only need 300 epoches to converge.

This  is the last log of net1 training,  it seems all right. And i sure my case name is correct when i train net2 using the case  name of net1

[0501 12:30:27 @monitor.py:459] QueueInput/queue_size: 50
[0501 12:30:27 @monitor.py:459] net1/train/acc: 0.86466
[0501 12:30:27 @monitor.py:459] net1/train/loss: 0.41775
[0501 12:30:27 @base.py:286] Training has finished!
2019-05-01 12:30:27.943574: W tensorflow/core/kernels/queue_base.cc:295] _0_QueueInput/input_queue: Skipping cancelled e
nqueue attempt with queue not closed
[0501 12:30:27 @input_source.py:174] EnqueueThread QueueInput/input_queue Exited.

**Comment by ArtemisZGL** [comment]
it's because i forgot to save the result of net1. After about 300 epoch, the net2 losses reached 0.006, and it's  quiet small at the beginning.

**Comment by Amey1109** [comment]
Can you provide me the pretrained weights?
